$NetBSD: patch-ae,v 1.3 2001/08/18 09:53:02 wiz Exp $

 1. Don't pollute G1 to allow 8-bit encondings.

 2. The original code make the TERMCAP env var
    longer than 1023 bytes, which cause problems.
    This is a quick hack but works with our termcap.

--- termcap.c.orig	2003-09-08 14:45:36.017301504 +0000
+++ termcap.c
@@ -76,7 +76,7 @@ static const char TermcapConst[] = "\\\n
 \t:DO=\\E[%dB:LE=\\E[%dD:RI=\\E[%dC:UP=\\E[%dA:bs:bt=\\E[Z:\\\n\
 \t:cd=\\E[J:ce=\\E[K:cl=\\E[H\\E[J:cm=\\E[%i%d;%dH:ct=\\E[3g:\\\n\
 \t:do=^J:nd=\\E[C:pt:rc=\\E8:rs=\\Ec:sc=\\E7:st=\\EH:up=\\EM:\\\n\
-\t:le=^H:bl=^G:cr=^M:it#8:ho=\\E[H:nw=\\EE:ta=^I:is=\\E)0:";
+\t:le=^H:bl=^G:cr=^M:it#8:ho=\\E[H:nw=\\EE:ta=^I:is=\\E(B:";
 
 char *
 gettermcapstring(s)
@@ -820,7 +820,7 @@ char *s;
 {
   register int n;
 
-  if (tcLineLen + (n = strlen(s)) > 55 && Termcaplen < TERMCAP_BUFSIZE - 4 - 1)
+  if (tcLineLen + (n = strlen(s)) > 69 && Termcaplen < TERMCAP_BUFSIZE - 4 - 1)
     {
       strcpy(Termcap + Termcaplen, "\\\n\t:");
       Termcaplen += 4;
